### ISSUE TITLE
The decorative image is not hidden from screen readers.

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#5788]
## Area or Component

Static Site 
- simplereport.gov

## Quick Summary
The use of a text alternative with decorative images is not appropriate. The decorative img is missing an empty alt attribute (alt="") or role="presentation".

### How to Replicate
Following images are decorative:

<img width="106" alt="Screenshot 2023-05-12 at 11 33 54" src="https://github.com/CDCgov/prime-simplereport/assets/20211771/35393ebe-4d2b-422f-8f1d-f0f30af8630c">

<img width="327" alt="Screenshot 2023-05-12 at 11 34 10" src="https://github.com/CDCgov/prime-simplereport/assets/20211771/617d2243-2736-403e-9311-764b5f71cc6d">

## Accessibility Violation

[[why is this inaccessible? include deque link if available.]: #
](https://docs.deque.com/issue-help/1.0.0/en/alt-text-decorative-inappropriate)

## Action Requested

Put empty `alt=""` attribute or `role="presentation"`

## Source Code (optional)

[which file or line is the issue attached to?]: #
